test/index.test.ts

```diff
@@ -461,12 +464,47 @@ test('async error', async () => {
     caughtError = e
   }
 
-  expect(spy.results[0][0]).toEqual('error')
-  expect(spy.results[0][1].message).toEqual('async error')
+  expect(spy.results).toEqual([['ok', promise]])
+  expect(spy.resolves[0][0]).toEqual('error')
+  expect(spy.resolves[0][1].message).toEqual('async error')
   expect(caughtError).toBeInstanceOf(Error)
   expect(caughtError?.message).toEqual('async error')
 })
 
+test.only('async order preserved', async () => {
```

Comment:
`.only` are not allowed, not sure why CI doesn't fail

Reply:
Woups. I missed it